code/modules/clothing/suits/marine_armor.dm

```diff
@@ -132,6 +132,44 @@
 	icon_state = "6"
 	armor = list("melee" = 30, "bullet" = 70, "laser" = 45, "energy" = 30, "bomb" = 60, "bio" = 10, "rad" = 10, "fire" = 30, "acid" = 30)
 
+/obj/item/clothing/suit/storage/marine/M3UL
+	name = "\improper M3-UL pattern marine armor"
+	desc = "A standard Marine M3 Ultralite Pattern Chestplate. Even lesser encumbrance and protection and cannot carry heavy weapons, but comes with a inbuilt motion detector. Provides high protection against acids and allows to be quickly taken on and off. Just be careful of machetes and bombs. Feels like nothing is here."
+	icon_state = "10"
+	armor = list("melee" = 15, "bullet" = 10, "laser" = 35, "energy" = 10, "bomb" = 5, "bio" = 0, "rad" = 0, "fire" = 5, "acid" = 70)
```

Comment:
70 acid armor seems like a bit too much, but we can test it regardless

Reply:
Anything higher than 30 would do enough.

Reply:
This is supposed to be an extremely light armor, giving it more acid protection than literally all other marine and PMC armors is a bit of a weird decision.

Reply:
I'll make it 50 then.

Reply:
The HEAVY marine armor has 30. the light 10, I think 20-25 would be the most fair.

Reply:
This needs adjustment